ResearchKit/Common/ORKImageCaptureView.m

```diff
@@ -197,31 +221,15 @@ - (void)setSession:(AVCaptureSession * __nullable)session {
     [self orientationDidChange];
 }
 
-- (UIBarButtonItem *)continueButtonItem {
-    return _continueSkipContainer.continueButtonItem;
-}
-
-- (void)setContinueButtonItem:(UIBarButtonItem *)continueButtonItem {
-    // Intercept the continue button press.  This can be called multiple
-    // times with the same UIBarButtonItem, or with different UIBarButtonItems,
-    // so capture it whenever the button does not point to our selector
-    if (continueButtonItem.action != @selector(capturePressed)) {
-        self.continueAction = continueButtonItem.action;
-        self.continueTarget = continueButtonItem.target;
-        self.continueTitle = continueButtonItem.title;
-        continueButtonItem.action = @selector(capturePressed);
-        continueButtonItem.target = self;
-    }
+- (void)setSkipButtonItem:(UIBarButtonItem *)skipButtonItem {
+    if (_showSkipButtonItem) {
+        _skipButtonItem = skipButtonItem;
     
-    // If we do not have an error to show, and we haven't already gotten a captured
-    // image, then change the title of the button to be appropriate for the capture action
-    if (!self.capturedImage && !self.error) {
-        continueButtonItem.title = ORKLocalizedString(@"CAPTURE_BUTTON_CAPTURE_IMAGE", nil);
-    } else if(self.capturedImage) {
-        _continueSkipContainer.skipButtonItem = _skipButtonItem;
+        // If the recapture button is not currently being used as the skip button, then use this new button
+        if (_continueSkipContainer.skipButtonItem != _recaptureButtonItem) {
+            _continueSkipContainer.skipButtonItem = skipButtonItem;
+        }
     }
-    
-    _continueSkipContainer.continueButtonItem = continueButtonItem;
 }
 
```

Comment:
Is `_showSkipButtonItem` necessary here? I think it should leave ORKStepViewController to handle `optional`. (StepVC will set nil or leave it unset)

And even if a step is not optional, a skip button can still be set by application, and we need to make it possible.

I think this would just work:
```
- (void)setSkipButtonItem:(UIBarButtonItem *)skipButtonItem {
        _skipButtonItem = skipButtonItem;
        // If the recapture button is not currently being used as the skip button, then use this new button
        if (_continueSkipContainer.skipButtonItem != _recaptureButtonItem) {
            _continueSkipContainer.skipButtonItem = skipButtonItem;
        }
}
```

Similar to skipButtonItem, it is also nice to have a setter for continueButtonItem:
```
- (void) setContinueButtonItem:(UIBarButtonItem *) continueButtonItem {
        _continueButtonItem = continueButtonItem;
        // If the recapture button is not currently being used as the skip button, then use this new button
        if (_continueSkipContainer.continueButtonItem != _captureButtonItem) {
            _continueSkipContainer.continueButtonItem = continueButtonItem;
        }
}
```